adflow/pyADflow.py

```diff
@@ -5649,6 +5749,52 @@ def _createFortranStringArray(self, strList):
 
         return arr
 
+    def _convertListToFortranStringArray(self, strList):
+        """Setting arrays of strings in Fortran can be kinda nasty. This
+        takes a list of strings and returns the array"""
```

Comment:
What's the difference between this function and `_createFortranStringArray`? The docstring is the same

Reply:
This new function is more robust. I thought I'd include it at the same time as the inverse function, `_convertFortranStringArrayToList`.
I'll use it in future PRs. But I guess it can wait for now.

Reply:
If the new function is more robust, I would be fine with completely replacing the old function. This is fine too though, we can add it later.